### PR TITLE
added storybook and component for heading

### PR DESCRIPTION
**Pull request by Andreas-sjolund-chas**
* added storybook - Heading.js
* added component - Heading.js

Heading accepts two parameters: "type" and "color"

ex. `<Heading type"h1" color="secondary">SOME TEXT HERE</Heading>`

### CODE REVIEW

**Comment by Ciavarella** [comment]
This looks really good @Andreas-sjolund-chas !

**Comment by robertjarske** [comment]
Looks good @Andreas-sjolund-chas

File: src/Elements/Heading.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { css, withStyles } from "../withStyles";
+
+const Heading = ({ type, color = "default", styles, ...props }) => {
+  const HeadingTag = `${type}`;
```
Comment (Andreas-sjolund-chas) [inline src/Elements/Heading.js]: Might change this to `h${size}` instead of `${type} for more consistency between components and prevent confusion.

**Comment by lessp** [comment]
Closes #2

**Pull request merged**